src/Serializer/src/Bootloader/SerializerBootloader.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Spiral\Serializer\Bootloader;
+
+use Spiral\Boot\Bootloader\Bootloader;
+use Spiral\Boot\Environment;
+use Spiral\Config\ConfiguratorInterface;
+use Spiral\Core\Container;
+use Spiral\Core\Container\Autowire;
+use Spiral\Serializer\Config\SerializerConfig;
+use Spiral\Serializer\Serializer\JsonSerializer;
+use Spiral\Serializer\Serializer\PhpSerializer;
+use Spiral\Serializer\SerializerInterface;
+use Spiral\Serializer\SerializerRegistry;
+use Spiral\Serializer\SerializerManager;
+use Spiral\Serializer\SerializerRegistryInterface;
+
+final class SerializerBootloader extends Bootloader
+{
+    protected const SINGLETONS = [
+        SerializerManager::class => [self::class, 'initSerializerManager'],
+        SerializerRegistryInterface::class => [self::class, 'initSerializerRegistry'],
```

Comment:
Maybe add additional singleton for SerializerInterface?

```
SerializerInterface::class => SerializerManager::()/
```

Reply:
Added